src/main/java/net/sacredlabyrinth/Phaed/PreciousStones/managers/SettingsManager.java

```diff
@@ -569,6 +568,8 @@ public boolean isRedstone(Block block) {
         		block.getType().equals(Material.JUKEBOX) ||
         		block.getType().equals(Material.LEVER) ||
                 block.getType().equals(Material.REPEATER) ||
+                block.getType().equals(Material.DAYLIGHT_DETECTOR) ||
```

Comment:
Fixes #51

Reply:
Nice thanks! I may have some time tomorrow to check the permission system you made